### DISCUSSION TITLE
no prompt to authorize

### DISCUSSION THREAD

**Discussion by eleach** [community/community#7520] [Copilot Conversations] [answered]
I am on vscode 1.62.0 on Ubuntu Linux.

The extension says it is enabled but I am not being prompted to authorize.

Thanks.

**Comment by PaulMorel** [comment]
I'm on VSCode 1.62.3 on Windows 10 and I don't get the prompt either. There doesn't seem to be a command to do so either.

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
